src/components/Errors/Errors.tsx

```diff
@@ -0,0 +1,41 @@
+import classNames from 'classnames';
+
+type Props = {
+  error:string;
+  closeError: () => void;
+};
+
+export const Errors: React.FC<Props> = (
+  {
+    error,
+    closeError,
+  },
+) => {
+  return (
+    <>
+      {error && (
+        <div
+          data-cy="ErrorNotification"
+          className={classNames(
+            'notification is-danger is-light has-text-weight-normal',
+            {
+              hidden: error.length === 0,
+            },
+          )}
+        >
+          <button
+            data-cy="HideErrorButton"
+            type="button"
+            className="delete"
+            aria-label="HideErrorButton"
+            onClick={() => closeError()}
+          />
+
```

Comment:
Create Notifcication component and pass to it type for exmple error or succsess and text so you will be able to reuse it in different cases